experimental/packages/opentelemetry-sdk-node/test/sdk.test.ts

```diff
@@ -850,6 +894,152 @@ describe('Node SDK', () => {
       await sdk.shutdown();
     });
   });
+
+  describe('configuring logger provider from env', () => {
+    let stubLogger: Sinon.SinonStub;
+
+    beforeEach(() => {
+      stubLogger = Sinon.stub(diag, 'info');
+    });
+
+    afterEach(() => {
+      stubLogger.reset();
+    });
+
+    it('should log a warning if OTEL_LOGS_EXPORTER not set', async () => {
```

Comment:
I kinda think this test case should be dropped. If the NodeSDK supports `OTEL_LOGS_EXPORTER` and the spec says that the default value is `otlp`, then by default the NodeSDK should export logs via OTLP and no need to *warn* about it.